benchmarks/ClusterBenchmark/Program.cs

```diff
@@ -119,7 +120,7 @@ private static void RunFireForgetClient()
                         var id = "myactor" + rnd.Next(0, ActorCount);
                         semaphore.Wait(() => {
                                 return cluster.RequestAsync<HelloResponse>(id, "hello", new HelloRequest(),
-                                    new CancellationTokenSource(TimeSpan.FromSeconds(5)).Token
+                                    CancellationTokens.WithTimeout(20*1000)
```

Comment:
Extend timeout in cluster benchmarks to allow for retries inside the IdentityWorker.
If the request times out faster than the identity worker retries, you get null response